baselines/teams.md

```diff
@@ -385,7 +385,7 @@ All the settings in this section are configured in the **Microsoft Teams admin c
 
 2.  Select **Teams** > **Teams Settings**.
 
-3.  Under the **Email integration** section, set **Allow users to send
+3.  Under the **Email integration** section, set **Users can send
```

Comment:
For MS.TEAMS.2.3v1, the note about not being available in GCC is located with the policy.  For this one, it is in license requirements.  Suggest being consistent across the two.  I prefer the policy locaton as a note vs. license for this, reserving license for 3 vs. 5.

Reply:
Updated to make licensing and region applicability (GCC, GCC High, DoD) consistent across policies.